target_redshift/db_sync.py

```diff
@@ -644,7 +659,8 @@ def load_csv(self, s3_key, count, size_bytes, compression=False, manifest=False)
                             FROM {stage_table} stage
                             JOIN {target_table} target
                             ON {join_condition}
-                            AND stage._sys_updated_at > target._sys_updated_at
+                            AND (stage._sys_log_poisition > target._sys_log_position AND
```

Comment:
Looks like there's a typo, _sys_log_poisition is mispelled.

Reply:
Nice catch yea I figure when I ran it it'd blurp but yea will fix thanks!